apps/web/src/pages/templates/workflow/DigestMetadata.tsx

```diff
@@ -11,50 +13,60 @@ export const DigestMetadata = ({ control, index }) => {
 
   return (
     <>
-      <Controller
-        control={control}
-        name={`steps.${index}.metadata.unit`}
-        render={({ field }) => {
-          return (
-            <Select
-              disabled={readonly}
-              error={errors?.steps ? errors.steps[index]?.metadata?.unit?.message : undefined}
-              label="Time unit"
-              placeholder="Time unit"
-              data={[
-                { value: DigestUnit.SECONDS, label: 'Seconds' },
-                { value: DigestUnit.MINUTES, label: 'Minutes' },
-                { value: DigestUnit.HOURS, label: 'Hours' },
-                { value: DigestUnit.DAYS, label: 'Days' },
-              ]}
-              data-test-id="time-unit"
-              {...field}
+      <InputWrapper label="Time Interval" description="Set the time intervals for the batch" styles={inputStyles}>
```

Comment:
Maybe worth adding front end tests for this aswell to make sure data is saved correctly and displayed on edit.